components/Extensions/samples/AttachedShadows.md

```diff
@@ -1,20 +1,18 @@
 ---
-title: Attached Shadows
+title: Shadows overview
 author: michael-hawker
 description: Attached Shadows allow you to easily create shadow effects on elements.
 keywords: shadow, shadows, dropshadow, dropshadowpanel, attachedshadow, attacheddropshadow, attachedcardshadow
 dev_langs:
   - csharp
 category: Extensions
-subcategory: Media
+subcategory: Shadows
 discussion-id: 0
 issue-id: 0
 icon: Assets/Shadow.png
 ---
 > **Platform APIs:** [`AttachedCardShadow`](/dotnet/api/microsoft.toolkit.uwp.ui.media.attachedcardshadow), [`AttachedDropShadow`](/dotnet/api/microsoft.toolkit.uwp.ui.attacheddropshadow)
 
-## Introduction
-
 There are two types of attached shadows available today, the `AttachedCardShadow` and the `AttachedDropShadow`. It is recommended to use the `AttachedCardShadow` where possible, if you don't mind the dependency on Win2D. The `AttachedCardShadow` provides an easier to use experience that is more performant and easier to apply across an entire set of elements, assuming those elements are rounded-rectangular in shape. The `AttachedDropShadow` provides masking support and can be leveraged in any UWP app without adding an extra dependency.
 
 ### Capability Comparison
```

Comment:
These should be ## two-level headers.

Thinking the docs validation is going to yell at us for not having titles at the top, but I can inject the title from the metadata in on the transpose step...